### DISCUSSION TITLE
Error after downloading the video

### DISCUSSION THREAD

**Discussion by Adeeb0** [aclap-dev/video-downloadhelper#427] [Help and Q&A]
### Your question

When the download is finished, this error appears

![13](https://github.com/aclap-dev/video-downloadhelper/assets/152232292/e2854984-06e0-414c-bbb1-387dd24a5fcf)

The video is fully downloadable, but it does not work, neither sound nor image, as shown in the picture

![12](https://github.com/aclap-dev/video-downloadhelper/assets/152232292/0a0ec80c-f470-46f3-bbd0-a439c44538d1)

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: ar
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Asus",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
Looks like DRM protection from that picture, nothing can be done about it.

**Reply by Adeeb0** [reply]
thank you
